lib/modules/manager/git-submodules/extract.spec.ts

```diff
@@ -13,7 +13,7 @@ import { extractPackageFile } from '.';
 
 jest.mock('simple-git');
 const simpleGit: jest.Mock<Partial<SimpleGit>> = _simpleGit as never;
-// eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
+
```

Comment:
why this change?

Reply:
lint-fix automatic cuz of new version

Reply:
I have switched this to target the v33 branch just now. Not sure if that will make any difference. This change is strange and probably something is wrong

Reply:
i got 4 lint errors when i ran `yarn lint` and it was the same here in github checks, i can delete them but lint will fail :/